hydra-main/src/main/java/com/addthis/hydra/job/spawn/Spawn.java

```diff
@@ -1932,6 +1948,7 @@ public void stopJob(String jobUUID) throws Exception {
     public void killJob(String jobUUID) throws Exception {
         boolean success = false;
         while (!success && !shuttingDown.get()) {
+            // Maybe
```

Comment:
I think this loop is concerning for an unrelated reason -- we call tryLock over and over again without any sort of delay in between. Maybe we should sleep for a few millis?